### PR TITLE
Update documentation on assets to indicate how to add new files

### PR DESCRIPTION
**Pull request by AWegnerGitHub**

### CODE REVIEW

File: content/Elegant - Pelican Theme/assets-plugin.md [hunk 1]
```diff
@@ -4,9 +4,9 @@ Category: Elegant - Pelican Theme
...
 Disqus_identifier: hk9m5eq-avoid-unnecessary-http-requests
-Subtitle: 
-Summary: Use Pelican assets plugin to improve your website load speed 
-Keywords: 
+Subtitle:
+Summary: Use Pelican assets plugin to improve your website load speed
+Keywords:
 
 Visitor's browser will make separate HTTP requests to fetch `elegent.css`,
 `custom.css`, `pygments.css` and `tipuesearch.css`. These four separate
```
Comment (talha131) [inline content/Elegant - Pelican Theme/assets-plugin.md]: Now we have admonition.css too

Comment (talha131) [inline content/Elegant - Pelican Theme/assets-plugin.md]: Not four. Five.

File: content/Elegant - Pelican Theme/assets-plugin.md [hunk 2]
```diff
@@ -26,6 +26,14 @@ Then enable `assets` plugin in your configuration.
 This minor fix will improve the load speed of your website. All four style
 sheets will be merged and minified into one style sheet, `style.min.css`.
 
+If you add a new CSS file to the theme while developing the Elegant theme, you
```
Comment (talha131) [inline content/Elegant - Pelican Theme/assets-plugin.md]: Lets move this patch to the end. And add heading "Hacking Elegant Source Code"

Comment (AWegnerGitHub) [reply content/Elegant - Pelican Theme/assets-plugin.md]: It doesn't look like github noticed this one, but I've added the new section.

**Pull request merged**